src/Datadog.Trace/Tracer.cs

```diff
@@ -361,6 +366,130 @@ private static IStatsd CreateDogStatsdClient(TracerSettings settings, string ser
             return new Statsd(statsdUdp, new RandomGenerator(), new StopWatchFactory(), prefix: string.Empty, constantTags);
         }
 
+        private static void StartStandaloneAgentProcessesWhenConfigured()
+        {
+            try
+            {
+                var datadogConfigPath = Environment.GetEnvironmentVariable(ConfigurationKeys.AgentConfigPath);
+                string traceProcessArgs = null;
+                string dogStatsDArgs = null;
+
+                if (!string.IsNullOrWhiteSpace(datadogConfigPath))
+                {
+                    traceProcessArgs = $"--config \"{datadogConfigPath}\"";
+                }
+
+                _traceAgentMonitor = StartProcessWithKeepAlive(ConfigurationKeys.AgentPath, traceProcessArgs);
+                _dogStatsDMonitor = StartProcessWithKeepAlive(ConfigurationKeys.DogStatsDPath, dogStatsDArgs);
+            }
+            catch (Exception ex)
+            {
+                DatadogLogging.RegisterStartupLog(log => log.Error(ex, "Error when attempting to start standalone agent processes."));
+            }
+        }
+
+        private static bool ProgramIsRunning(string fullPath)
+        {
+            if (string.IsNullOrWhiteSpace(fullPath))
+            {
+                return false;
+            }
+
+            var fileName = Path.GetFileNameWithoutExtension(fullPath).ToLower();
```

Comment:
`Process.GetProcessesByName()` is not case-sensitive.
```suggestion
            var fileName = Path.GetFileNameWithoutExtension(fullPath);
```